### PR TITLE
ZH-trans: add test.md

### PR DESCRIPTION
**Pull request by xichengliudui**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/test.md [hunk 1]
```diff
@@ -0,0 +1,706 @@
...
+<!--
+title: Docs smoke test page
+main_menu: false
+-->
+
+<!--
+This page serves two purposes:
+-->
+此页面有两个用途：
+
```
Comment (stewart-yu) [inline content/zh/docs/test.md]: 文档？

**Comment by fleeto** [comment]
`[trans-bot:N/A]`

Thank you @xichengliudui, the [related task issue](https://github.com/k8smeetup/website-tasks/issues/651) has no assignee. will not be reported to the task issues

File: content/zh/docs/test.md (hunk 1 above)
Comment (chenrui333) [reply content/zh/docs/test.md]: 我觉得 文档也行，不必拘泥于 `page`

File: content/zh/docs/test.md [hunk 2]
```diff
@@ -0,0 +1,706 @@
...
+-->
+- 这是一个新列表。使用 Hugo，您需要使用 HTML 注释来分隔两个连续的列表。**HTML 注释需要在左边的空白处。**
+- 项目符号列表可以包含段落或块元素。
+
+<!--
+      Indent the content to be one tab stop beyond the text of the bullet
+      point. **This paragraph and the code block line up with the second `l` in
+      `Bullet` above.**
+-->
+      将内容缩进到项目符号文本之外的一个制表符停止。**本段和代码块与上面 `Bullet` 中的第二个 `l` 对齐。**
```
Comment (xiangpengzhao) [inline content/zh/docs/test.md]: ”停止“ 是不是可以去掉？`tab stop`整体应该就是表示”制表符“了

File: content/zh/docs/test.md [hunk 3]
```diff
@@ -0,0 +1,706 @@
...
+3.  {{<note>}}
+    For single-digit numbered lists, using two spaces after the period makes
+    interior block-level content line up better along tab-stops.
+    {{</note>}}
+-->
+
+1.  这是一个列表项
+2.  这是同一列表中的另一个列表项。在 Markdown 中使用的数字不一定与最终输出中的数字相关。按照惯例，我们使它们保持同步。
+3.  {{<note>}}
+    对于单位数字编号列表，在句点后使用两个空格可使内部块级内容沿制表位更好地排列。
```
Comment (xiangpengzhao) [inline content/zh/docs/test.md]: ”制表位“ 最好和全文其他地方统一，使用“制表符”

File: content/zh/docs/test.md [hunk 4]
```diff
@@ -0,0 +1,706 @@
...
+
+{{< warning >}}
+
+<!--
+There is one situation where you need to use indentation for code blocks: when
+the contents of the code block contain lines starting with `-` or `*` characters.
+This is due to
+[blackfriday issue #239](https://github.com/russross/blackfriday/issues/239).
+-->
+有一种情况需要对代码块使用缩进：当代码块的内容包含以 `-` 或 `*` 字符开头的行时。这是由于 [blackfriday 问题 #239(https://github.com/russross/blackfriday/issues/239).
```
Comment (xiangpengzhao) [inline content/zh/docs/test.md]: 漏掉了 `]`

File: content/zh/docs/test.md [hunk 5]
```diff
@@ -0,0 +1,706 @@
...
+-->
+要格式化链接，请将链接文本放在方括号中，然后将链接目标放在圆括号中。[链接到 Kubernetes.io](https://kubernetes.io/) 或者 
+[Kubernetes.io 的相关链接](/)
+
+<!--
+You can also use HTML, but it is not preferred.
+<a href="https://kubernetes.io/">Link to Kubernetes.io</a>
+-->
+您也可以使用 HTML，但不推荐使用它。
+<a href="https://kubernetes.io/">连接到 Kubernetes.io</a>
```
Comment (xiangpengzhao) [inline content/zh/docs/test.md]: 链接

File: content/zh/docs/test.md [hunk 6]
```diff
@@ -0,0 +1,706 @@
...
+>
+> You can even have code blocks.
+>
+> ```bash
+> sudo dmesg
+> ```
+>
+-->
+
+> 这是一个侧边栏。
```
Comment (xiangpengzhao) [inline content/zh/docs/test.md]: 翻译的内容不要注释掉

File: content/zh/docs/test.md [hunk 7]
```diff
@@ -0,0 +1,706 @@
...
+
+<!--
+| Heading cell 1 | Heading cell 2 |
+|----------------|----------------|
+| Body cell 1    | Body cell 2    |
+-->
+
+| 标题单元 1     | 标题单元 2    |
+|---------------|----- ---------|
+| 身体单元 1     | 身体单元 2    |
```
Comment (xiangpengzhao) [inline content/zh/docs/test.md]: 这里的 body 翻译成“主体”比较合适，下同

File: content/zh/docs/test.md [hunk 8]
```diff
@@ -0,0 +1,706 @@
+---
+title: 文档 smoke 测试页面
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply content/zh/docs/test.md]: 👍

File: content/zh/docs/test.md [hunk 9]
```diff
@@ -0,0 +1,706 @@
...
+- ~~strikethrough~~
+- <u>underline</u>
+- _<u>underline italic</u>_
+- **<u>underline bold</u>**
+- ***<u>underline bold italic</u>***
+- `monospace text`
+- **`monospace bold`**
+-->
+
+- **bold**
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply content/zh/docs/test.md]: 👍

File: content/zh/docs/test.md [hunk 10]
```diff
@@ -0,0 +1,706 @@
...
+  within a list item an extra 4 spaces too.
+
+- To end a list and start another, you need a HTML comment block on a new line
+  between the lists, flush with the left-hand border. The first list won't end
+  otherwise, no matter how many blank lines you put between it and the second.
+-->
+
+- 确保缩进子列表项**4个空格**而不是您可能习惯的2个空格。与直觉相反，您需要将列表项中的块级内容缩进额外的4个空格。
+- 要结束列表并启动另一个列表，您需要在列表之间的新行上使用 HTML 注释块，并使用左侧边框刷新。
+  无论你在第二个列表和第二个列表之间放置多少空行，第一个列表都不会以其他方式结束。
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply content/zh/docs/test.md]: 😆

File: content/zh/docs/test.md (hunk 6 above)
Comment (xichengliudui) [reply content/zh/docs/test.md]: `>` 并不是注释。

File: content/zh/docs/test.md [hunk 11]
```diff
@@ -0,0 +1,707 @@
...
+      point. **This paragraph and the code block line up with the second `l` in
+      `Bullet` above.**
+-->
+      将内容缩进到项目符号文本之外的一个制表符。**本段和代码块与上面 `Bullet` 中的第二个 `l` 对齐。**
+
+      ```bash
+      ls -l
+      ```
+
+      - 以及一些块级内容之后的子列表
```
Comment (xichengliudui) [inline content/zh/docs/test.md]: And a sub-list after some block-level content

**Pull request merged**